client/api/events.ts

```diff
@@ -176,7 +176,7 @@ function create(updates: Partial<IEventItem>): Promise<Array<IEventItem>> {
 function update(original: IEventItem, updates: Partial<IEventItem>): Promise<Array<IEventItem>> {
     return superdeskApi.dataApi.patch<IEventItem>('events', original, {
         ...updates,
-        associated_plannings: undefined,
+        associated_plannings: updates.associated_plannings,
```

Comment:
Hey @thecalcc 
`associated_plannings` is set to `undefined` because we don't want to store this on the Event item itself (this happens for both create and update actions).

Instead what I would try is to re-populate this field after the response from the server/update action:

https://github.com/superdesk/superdesk-planning/blob/f69090523b4fc179c82d36dfa0086fb984cbb38c/client/api/events.ts#L199-L202

```ts
.then((response) => { 
    const events = modifySaveResponseForClient(response);
    
    events[0].associated_plannings = updates.associated_plannings
    return planningApi.planning.searchGetAll({ 
```

Can you please try that and see if that fixes the issue